app/components/ExamPages/SeventhChordsNotateTemplate.tsx

```diff
@@ -32,13 +37,16 @@ export default function NotateSeventhChords({
 
   useEffect(() => {
     currentUserDataRef.current = currentUserData;
+   // console.log("currentUserData: ", currentUserData);
   }, [currentUserData]);
 
   const handleSubmit = async (e: FormEvent) => {
     e.preventDefault();
     memoizedSetCurrentUserData({
       ...currentUserDataRef.current,
-      [seventhChordsPropName]: chords,
+      [seventhChordsPropName]: seventhChords,
+      [seventhChordDataPropName]: seventhChordData,
+      [seventhChordStavesPropName]: seventhChordStaves,
     });
     // console.log({
     //   ...currentUserDataRef.current,
```

Comment:
On line 108, NotateChord has errors because the component was changed and it is now expecting the following arguments: chordStaves, setChordStaves, setChords,